Source/WebCore/html/HTMLTableCellElement.cpp

```diff
@@ -110,7 +110,7 @@ void HTMLTableCellElement::collectPresentationalHintsForAttribute(const Qualifie
     switch (name.nodeName()) {
     case AttributeNames::nowrapAttr:
         addPropertyToPresentationalHintStyle(style, CSSPropertyWhiteSpaceCollapse, CSSValueCollapse);
-        addPropertyToPresentationalHintStyle(style, CSSPropertyTextWrap, CSSValueNowrap);
+        addPropertyToPresentationalHintStyle(style, CSSPropertyTextWrapMode, CSSValueNowrap);
```

Comment:
Just confirming that we do not need to also set `text-wrap-style` here?

Reply:
Same here